pulpcore/app/settings.py

```diff
@@ -28,19 +27,20 @@
 
 ALLOWED_HOSTS = ["*"]
 
-MEDIA_ROOT = "/var/lib/pulp/"
+DEPLOY_ROOT = Path("/var/lib/pulp")
+MEDIA_ROOT = DEPLOY_ROOT / "media"
```

Comment:
Offline discussion was to change this to `artifact` (which already exists) and then add a migration that updates the database.
```suggestion
MEDIA_ROOT = DEPLOY_ROOT / "artifact"
```

That means the following code needs updates:
https://github.com/pulp/pulpcore/blob/09d05da6f38c74e3574d3d256f890b23d76cb3d0/pulpcore/app/models/fields.py#L47-L66

Before proceeding with this, I'd like to get some more feedback if this is really desired because it looks like the entire `ArtifactFileField.pre_save` becomes redundant then.

Another approach would be to use a migration that looks at all models with a `FileField`. Then if it doesn't exist, look a directory higher and move it into `MEDIA_ROOT` https://github.com/django-extensions/django-extensions/blob/d747e09885e90d4c5f702f0bb4f0ea42e63f4eb0/django_extensions/management/commands/unreferenced_files.py#L27-L43 can be the basis of that.